monoio/src/driver/legacy/mod.rs

```diff
@@ -28,13 +33,13 @@ pub(crate) use waker::UnparkHandle;
 pub(crate) struct LegacyInner {
     pub(crate) io_dispatch: Slab<ScheduledIo>,
     #[cfg(unix)]
-    events: mio::Events,
+    events: Events,
     #[cfg(unix)]
     poll: mio::Poll,
     #[cfg(windows)]
-    events: iocp::Events,
+    events: Vec<Event>,
     #[cfg(windows)]
-    poll: iocp::Poller,
+    poll: Poller,
```

Comment:
I have 2 questions:
1. Which way do you want to iocp? It can be used as epoll to just get notified if the fd is ready, it can also be used like iouring to let it do io directly.
2. If it is designed to work as a readiness poller, can we use `mio::Poll` instead of brings a new poller impl?

Reply:
> 1. Which way do you want to iocp? It can be used as epoll to just get notified if the fd is ready, it can also be used like iouring to let it do io directly.

Just epoll.

> 2. If it is designed to work as a readiness poller, can we use `mio::Poll` instead of brings a new poller impl?

No, cause https://github.com/bytedance/monoio/issues/166#issuecomment-1605282671